### PR TITLE
Features/manual poke coin collection add functions and timers #1757

### PR DESCRIPTION
**Pull request by Furtif**
#1757

### CODE REVIEW

File: PoGo.Necrobot.Window/Model/PlayerInfoModel.cs [hunk 1]
```diff
@@ -9,11 +9,18 @@
...
+using POGOProtos.Inventory.Item;
+using System.Timers;
 
 namespace PoGo.Necrobot.Window.Model
 {
     public class PlayerInfoModel : ViewModelBase
     {
+        public DateTime Insence_expires = new DateTime(0);
+        public DateTime Luckyeggs_expires = new DateTime(0);
+        private System.Timers.Timer tmr;
```
Comment (CDAGaming) [inline PoGo.Necrobot.Window/Model/PlayerInfoModel.cs]: I don't think System.Tine needs to be referenced twice

File: PoGo.Necrobot.Window/Model/PlayerInfoModel.cs [hunk 2]
```diff
@@ -189,6 +196,48 @@ public void OnInventoryRefreshed(IEnumerable<InventoryItem> inventory)
...
+                            break;
+                    }
+                }
+            }
+
+            // add and enable timer
+            // 
+            // tmr
+            // 
+            tmr = new System.Timers.Timer()
```
Comment (CDAGaming) [inline PoGo.Necrobot.Window/Model/PlayerInfoModel.cs]: If Using System; is defined at the start, we don't need to reference it again

Meaning Just use Timers.Timer() instead

File: PoGo.Necrobot.Window/Model/PlayerInfoModel.cs [hunk 3]
```diff
@@ -265,5 +314,30 @@ public void UpdateEggs(double kmremaining)
...
+            else
+            {
+                // my value here  00:00:00
+                InsenceTMR = $"{time.Minutes}m {Math.Abs(time.Seconds)}s";
+            }
+
+            var timel = Luckyeggs_expires - DateTime.UtcNow;
+            if (Luckyeggs_expires.Ticks == 0 || timel.TotalSeconds < 0)
+            {
+                //not implanted
```
Comment (CDAGaming) [inline PoGo.Necrobot.Window/Model/PlayerInfoModel.cs]: Implemented*

**Comment by Furtif** [comment]
@CDAGaming Control it, I did it like that all of a sudden lol

Miss off the scrollbar of the incense items, lucky egg on the itemslist, since you can not use more than one ... et maybe change the "drop" button by the timer and deactivate button until the time runs out..

**Comment by CDAGaming** [comment]
@Furtif I'm around 3 hours from home atm, but when I'm back I'll see the changes and see if I can optimize the button layout. Will see on My requested changes to

**Comment by Furtif** [comment]
It will merge this branch to feature... I'm going to eat something....

**Comment by Furtif** [comment]
https://ci.appveyor.com/project/jjskuld/necrobot/build/0.0.0.1773-ibninnba ok 👍

**Pull request merged**